api/specs/web-server/_resource_usage.py

```diff
@@ -93,7 +96,7 @@ async def list_resource_usage_services(
 
 @router.get(
     "/services/-/aggregated-usages",
-    response_model=Envelope[list[ServiceRunGet]],
+    response_model=Envelope[list[OsparcCreditsAggregatedByServiceGet]],
```

Comment:
You are exposing one-to-one a model from an internal API to the web-API. NOte that the latter has some conventions e.g. expected camel-case. 
In your case is simple to solve. You can use e.g. this https://github.com/itisfoundation/osparc-simcore/blob/0cc540f3156bda9c8690a2b10f310aeee6a24afa/packages/models-library/src/models_library/api_schemas_webserver/catalog.py#L233-L235


The problem is when you have embedded models and you want to guarantee e.g. that all fields are camelcase.

I know it is inconvenient (i.e. APITA) but we either :
- use one-word field names
- or drop this convention in the web-api and use only lower case (my preferred one)
- or enforce camelcase only in the web-api bodies/paylods. We should add some tests if we really want to enforce these conventions.

Reply:
Yes, I agree with you, and you are correct. I guess I missed this when I was doing it last minute before the last review, which is why the `ServiceRunGet` was completely wrong here.